site-src/implementations.md

```diff
@@ -162,13 +162,13 @@ APISIX currently supports Gateway API `v1beta1` version of the specification for
 
 [Avi Kubernetes Operator (AKO)][ako] provides L4-L7 load-balancing using VMware NSX Advanced Load Balancer.
```

Comment:
@pkoshtavmware please change `... VMware NSX Advanced Load Balancer` to `... VMware AVI Load Balancer`